package.json

```diff
@@ -282,7 +282,6 @@
   },
   "devDependencies": {
     "@babel/parser": "^7.5.5",
-    "@babel/plugin-syntax-dynamic-import": "^7.2.0",
```

Comment:
Have you re-run `yarn kbn bootstrap` to update _yarn.lock_?

Reply:
Ah, good eye.

Both removed packages are still in the dependency tree because of `@kbn/storybook` -> `@storybook/react`. Versions matched as well, so no `yarn.lock` update occurred.